lib/criticalrunner.js

```diff
@@ -20,8 +20,9 @@ system.args sent from critical.js:
 		width = system.args[2],
 		height = system.args[3],
 		forceInclude = JSON.parse(system.args[4]),
-		rulesFile = system.args[5],
-		ignoreConsole = system.args.length > 6 && system.args[6] === "--ignoreConsole" ? true : false,
+		initialMap = JSON.parse(system.args[5]),
+		rulesFile = system.args[6],
+		ignoreConsole = system.args.length > 7 && system.args[7] === "--ignoreConsole" ? true : false,
```

Comment:
@jefflembeck 

Ah! That's what I wondered. Could I add this new param on to the end and open an issue for the next release to move it up the argument stack? That would make it a backward compatible feature and a minor release right?